examples/gamma/main.rs

```diff
@@ -50,21 +56,46 @@ const CLEAR_COLOR: [f32; 4] = [0.5, 0.5, 0.5, 1.0];
 
 type SurfaceData = <<ColorFormat as Formatted>::Surface as SurfaceTyped>::DataType;
 
+enum Backend {
```

Comment:
Why use a single variant enum instead of a tuple struct ?

Reply:
that came from my original attempt to make it a run-time switch. Fixed now